### PR TITLE
Add Brave Nightly version 91.1.28.59

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20111)

### CODE REVIEW

File: manifests/b/BraveSoftware/BraveBrowser-Nightly/91.1.28.59/BraveSoftware.BraveBrowser-Nightly.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+PackageIdentifier: BraveSoftware.BraveBrowser-Nightly
+PackageVersion: 91.1.28.59
```
Comment (ItzLevvie) [inline manifests/b/BraveSoftware/BraveBrowser-Nightly/91.1.28.59/BraveSoftware.BraveBrowser-Nightly.locale.en-US.yaml:2]:
```suggestion
PackageVersion: 92.1.28.59
```

File: manifests/b/BraveSoftware/BraveBrowser-Nightly/91.1.28.59/BraveSoftware.BraveBrowser-Nightly.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+PackageIdentifier: BraveSoftware.BraveBrowser-Nightly
+PackageVersion: 91.1.28.59
```
Comment (ItzLevvie) [inline manifests/b/BraveSoftware/BraveBrowser-Nightly/91.1.28.59/BraveSoftware.BraveBrowser-Nightly.installer.yaml:2]:
```suggestion
PackageVersion: 92.1.28.59
```

File: manifests/b/BraveSoftware/BraveBrowser-Nightly/91.1.28.59/BraveSoftware.BraveBrowser-Nightly.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+PackageIdentifier: BraveSoftware.BraveBrowser-Nightly
+PackageVersion: 91.1.28.59
```
Comment (ItzLevvie) [inline manifests/b/BraveSoftware/BraveBrowser-Nightly/91.1.28.59/BraveSoftware.BraveBrowser-Nightly.yaml:2]:
```suggestion
PackageVersion: 92.1.28.59
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Please update the version folder too.

**Review by OfficialEsco — APPROVED** [review]

**Comment by OfficialEsco** [comment]
Uhoh didnt see that D:

**Comment by vedantmgoyal2009** [comment]
@OfficialEsco I am creating a new PR :)

**Pull request merged**